app/assets/javascripts/libfigure/param.js

```diff
@@ -194,11 +194,16 @@ function stringParamSliceIncrement (value) {
 var param_slice_increment = {name: "slice increment", ui: chooser_slice_increment, value: 'couple', string: stringParamSliceIncrement};
 
 function stringParamDownTheHallEnder (value) {
-  if      ('' === value)             { return ''; }
-  else if ('turn-couples' === value) { return 'turn as couples'; }
-  else if ('turn-alone'   === value) { return 'turn alone'; }
-  else if ('circle'       === value) { return 'bend into a ring'; }
-  else if ('*'            === value) { return 'end however'; }
+  if      ('' === value)              { return ''; }
+  else if ('turn-couples'  === value) { return 'turn as couples'; }
+  else if ('turn-alone'    === value) { return 'turn alone'; }
+  else if ('circle'        === value) { return 'bend into a ring'; }
+  else if ('cozy'          === value) { return 'form a cozy line'; }
+  else if ('cloverleaf'    === value) { return 'bend into a cloverleaf';}
+  else if ('thread-needle' === value) { return 'thread the needle';}
+  else if ('right-high'    === value) { return 'right hand high, left hand low';}
+  else if ('sliding-doors' === value) { return 'go into a sliding doors movement';}
```

Comment:
My vote is `slide doors`. So the whole figure would print as "down the hall and slide doors".  It's even more parsimonious, and it's verblike.